src/Drivers/FTP.js

```diff
@@ -0,0 +1,200 @@
+const path = require('path')
+const JSFtp = require('jsftp')
```

Comment:
I believe this call need to be inside the constructor.

Example: https://github.com/Slynova-Org/node-flydrive/blob/develop/src/Drivers/AwsS3.js#L18